### PR TITLE
Added a command to make checkout of a script 

### PR DESCRIPTION
**Pull request by zLianK**
Added a command to make checkout of a script and a controller to check if a script is read-only.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
-# Change Log
-All notable changes to the "rech-editor-internal" extension will be documented in this file.
```
Comment (gustavosimon) [inline CHANGELOG.md:2]: I think we can keep this instructions in CHANGELOG.md. It's helpful to community contributors and we can know all the releases lifetime of this project.

What do you think @zLianK ?

File: src/wc/WorkingCopyScripts.ts [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+    /**
+     * Check if the file is inside the scripts directory (f:\bat)
+     *
+     * @param fileSystemPath File path to check
+     * @returns {boolean}
+     */
+    public static isScriptsDirectory(fileSystemPath: string): boolean {
+        if (window.activeTextEditor) {
+            const dir = new Path(fileSystemPath).directory().toLowerCase();
+            if (dir.includes("f:\\bat")) {
```
Comment (gustavosimon) [inline src/wc/WorkingCopyScripts.ts:22]: I think we should not write this path hardcoded, if this extension have to be public.
I know that have another classes that have hardcoded paths as here.
Maybe we can create settings to this params or make this Rech extension private.

What do you think @zLianK ?

Comment (zLianK) [reply src/wc/WorkingCopyScripts.ts:22]: I think we should make it private, because this is only for internal purpose.

File: CHANGELOG.md (hunk 1 above)
Comment (zLianK) [reply CHANGELOG.md:2]: I agree with you. I'll change this.

**Review by gustavosimon — APPROVED** [review]
LGTM

Can you merge @zLianK ?

**Pull request merged**